dataactcore/models/jobTrackerInterface.py

```diff
@@ -150,6 +150,13 @@ def markJobStatus(self,jobId,statusName):
         if prevStatus != 'finished' and statusName == 'finished':
             self.checkJobDependencies(jobId)
 
+    def getJobStatusNames(self):
+        """ Get All Job Status names """
+        job_status_names = []
+        for job_status in self.session.query(JobStatus.name):
+            job_status_names.append(str(job_status.name))
+        return job_status_names
+
```

Comment:
So this would be 2 lines, self.getJobStatusId(None) and return JobStatus.JOB_STATUS_DICT.keys()